src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -127,6 +127,9 @@ static ExperimentalFeature()
                 new ExperimentalFeature(
```

Comment:
@adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-linux`
Breakpoint SDK Unit Tests.Handling errors while managing breakpoints in a remote runspace via the SDK.Error occurred in Context block
```powershell
Exception calling "RemoveBreakpoint" with "1" argument(s): "The background process reported an error with the following message: Remote transport error: ."
At /home/vsts/work/1/s/test/powershell/SDK/Breakpoint.Tests.ps1:154 char:13
```